src/app/page/authorization/login/login.component.html

```diff
@@ -62,6 +62,22 @@
         <span>Your account successfully activated</span>
       </div>
     </div>
+    <div class="alert alert-danger" *ngIf="isBlocked">
+      <span>Account with this email is temporarily blocked by administrator.</span>
```

Comment:
You are not reusing the message that comes from server, maybe it doesn't worth to send it.

Reply:
Yes, you're right.